src/NexusMods.App/TelemetryProvider.cs

```diff
@@ -1,30 +1,55 @@
 using System.Reactive.Disposables;
+using DynamicData.Aggregation;
+using DynamicData.Binding;
 using Microsoft.Extensions.DependencyInjection;
+using NexusMods.Abstractions.FileStore.Downloads;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.Loadouts.Mods;
+using NexusMods.Abstractions.MnemonicDB.Attributes;
 using NexusMods.Abstractions.Telemetry;
 using NexusMods.App.UI;
 using NexusMods.Networking.NexusWebApi;
+using NexusMods.Paths;
 using OneOf;
 
 namespace NexusMods.App;
 
 internal sealed class TelemetryProvider : ITelemetryProvider, IDisposable
 {
-    private readonly CompositeDisposable _disposable;
+    private readonly CompositeDisposable _disposable = new();
+    private readonly IRepository<Loadout.Model> _loadoutRepository;
 
     public TelemetryProvider(IServiceProvider serviceProvider)
     {
-        _disposable = new CompositeDisposable();
+        _loadoutRepository = serviceProvider.GetRequiredService<IRepository<Loadout.Model>>();
 
+        // membership status
         var loginManager = serviceProvider.GetRequiredService<LoginManager>();
         loginManager.IsPremium.SubscribeWithErrorLogging(value => _isPremium = value).DisposeWith(_disposable);
+
+        // download size
+        var downloadAnalysisRepository = serviceProvider.GetRequiredService<IRepository<DownloadAnalysis.Model>>();
+        downloadAnalysisRepository
+            .Observable
+            .ToObservableChangeSet()
+            .Sum(x => (double)x.Size.Value)
+            .SubscribeWithErrorLogging(totalDownloadSize => _downloadSize = Size.From((ulong)totalDownloadSize))
+            .DisposeWith(_disposable);
     }
 
     public void ConfigureMetrics(IMeterConfig meterConfig)
     {
+#if RELEASE
+        meterConfig.CreateActiveUsersCounter(BuildInfo.ApplicationConstants.Version);
```

Comment:
Ugh, this is actually broken I think

Reply:
@erri120 Is that correct?

Reply:
Right, another case why preprocessor definitions suck.

Reply:
Agreed, I will fix it in my pr since I need to fix conflicts in this file anyways

Reply:
Pushed the fix as well